profile/run_whole.py

```
@@ -0,0 +1,293 @@
+from pathlib import Path
+from sorawm.core import SoraWM
+from sorawm.schemas import CleanerType
+from pathlib import Path
```

Ruff (0.14.7)

Redefinition of unused `Path` from line 1

Remove definition: `Path`

(F811)